htdocs/core/class/infobox.class.php

```diff
@@ -83,15 +83,15 @@ public static function getListOfPagesForBoxes()
 	/**
 	 *  Return array of boxes qualified for area and user
 	 *
-	 *  @param	DoliDB		$dbs				Database handler
+	 *  @param	DoliDB		$db				Database handler
 	 *  @param	string		$mode			'available' or 'activated'
 	 *  @param	int			$zone			Name or area (-1 for all, 0 for Homepage, 1 for Accountancy, 2 for xxx, ...)
 	 *  @param  User|null   $user	  		Object user to filter
 	 *  @param	array		$excludelist	Array of box id (box.box_id = boxes_def.rowid) to exclude
 	 *  @param  int         $includehidden  Include also hidden boxes
 	 *  @return array       	        	Array of boxes
 	 */
-	public static function listBoxes($dbs, $mode, $zone, $user = null, $excludelist = array(), $includehidden = 1)
+	public static function listBoxes($db, $mode, $zone, $user = null, $excludelist = array(), $includehidden = 1)
```

Comment:
Because the method is static, we used the var $dbs here to avoid conflict and overwritting global $db.

Reply:
@eldy ok thank you, i send a fix